src/core/Chip/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Icon from '../Icon'
+import s from './styles'
+
+class Chip extends React.PureComponent {
+    onRemove = event => {
```

Comment:
use succinct names `evt` instead of `event`

Reply:
4358c7d